src/main/kotlin/kweb/util/Template.kt

```diff
@@ -1,4 +1,4 @@
-package kweb.common
+package kweb.util
```

Comment:
Was there a reason for the package name change? Not opposed to it, just curious.

Reply:
I just randomly found it as I was clicking through the code. IntelliJ had it labeled as a warning, and I thought it made sense to change the package name to the package the file was actually in. It's more OCD than anything.

Reply:
Ah, didn't realize the package didn't match the dir.